02-peer-protocol.md

```diff
@@ -1022,6 +1221,7 @@ fee changes).
     to the ordering of the commitment transaction (see [BOLT #3](03-transactions.md#transaction-input-and-output-ordering)).
   - if it has not recently received a message from the remote node:
       - SHOULD use `ping` and await the reply `pong` before sending `commitment_signed`.
+  - MUST send a `commitsigs` for each splice in progress, in increasing feerate order.
```

Comment:
what does `in progress` mean here? a splice that is either waiting for negotiation to complete or is awaiting conf?

Reply:
It means splices waiting for confirmation.

Reply:
Going to make that more clear 👍